pkg/storage/db.go

```diff
@@ -9,9 +9,10 @@ import (
 	"sync"
 	"time"
 
-	"github.com/asecurityteam/asset-inventory-api/pkg/domain"
 	_ "github.com/lib/pq" // must remain here for sql lib to find the postgres driver
 	"github.com/pkg/errors"
+
+	"github.com/asecurityteam/asset-inventory-api/pkg/domain"
```

Comment:
Interesting. It seems like `goimports -local github.com/asecurityteam` now puts this on a separate section with a blank line before it. Does anyone have any comments/insight re: this?

Reply:
Possibly because this is an internal project module versus a public one?